sdk-springboot/src/main/java/io/dapr/springboot/domain/DaprBulkAppResponse.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2021 The Dapr Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package io.dapr.springboot.domain;
+
+/**
+ * Response from the application containing status for each entry in the bulk message.
+ * It is posted by the bulk subscribe handler.
+ */
+public class DaprBulkAppResponse {
+  public DaprBulkAppResponseEntry[] statuses;
```

Comment:
all fields should be private with get accessors only ...

Reply:
This domain object should be part of default SDK and not springboot ...

Reply:
Thanks, updated!